Language/Functions/Bits and Bytes/bitSet.adoc

```diff
@@ -34,7 +34,26 @@ Sets (writes a 1 to) a bit of a numeric variable.
 
 [float]
 === Returns
-Nothing
+x: the value of the numeric variable after the bit at position n is set.
+
```

Comment:
```suggestion

--
// OVERVIEW SECTION ENDS



// HOW TO USE SECTION STARTS
[#howtouse]
--
```

Put example in separate [block](https://docs.asciidoctor.org/asciidoc/latest/blocks/open-blocks/), per the standard reference page format:

https://github.com/arduino/reference-en/blob/074f4d2f081c9d6f7251b08b990b09709c3c580b/AsciiDoc_sample/Reference_Terms/AsciiDoc_Template-Single_Entity.adoc?plain=1#L52-L60